src/node.cc

```diff
@@ -829,11 +830,21 @@ static ExitCode InitializeNodeWithArgsInternal(
 
   HandleEnvOptions(per_process::cli_options->per_isolate->per_env);
 
+  std::string node_options;
+  auto file_path = node::Dotenv::GetPathFromArgs(*argv);
+
+  if (file_path.has_value()) {
+    auto cwd = Environment::GetCwd(Environment::GetExecPath(*argv));
+    std::string path = cwd + kPathSeparator + file_path.value();
+    CHECK(!per_process::v8_initialized);
+    auto dotenv = node::Dotenv();
+    dotenv.parse(path);
+    dotenv.assignNodeOptionsIfAvailable(&node_options);
```

Comment:
@nodejs/cpp-reviewers What is the correct way of accessing the environment `env_vars` (std::shared_ptr<kvstore>) in here? Afaik, it is not initialized yet, but I don't want to read the file again when the environment is initialized just to set the environment variables, and I couldn't find a good enough place to store the reference of this `dotenv` variable.